.github/workflows/pr-manual-merge.js

```diff
@@ -0,0 +1,71 @@
+const { slackNotification, getLocalConfigs } = require('./helpers.js');
+
+const LABELS = {
+  zeroImpact: 'zero-impact',
+};
+
+async function main({ github, context } = {}) {
+    if (!github || !context) {
+        throw new Error("GitHub context is missing. Ensure you are running in the correct environment.");
+    }
+    if (process.env.LOCAL_RUN) {
+        console.log("Local run detected. Loading local configurations...");
+        const { github: localGithub, context: localContext } = getLocalConfigs();
+        return main({ github: localGithub, context: localContext });
```

Comment:
You're running into a loop here, do the params not work?

1. you call main
2. it's a local run
3. you call main, as its a local run
4. its a local run
5. you call main as its a local run
6. its a local run
7. ....